src/Component/spec/Maker/Helper/GridHelperSpec.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace spec\Sylius\Component\Grid\Maker\Helper;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Grid\Maker\Helper\GridHelper;
+
+class GridHelperSpec extends ObjectBehavior
+{
+    function let(): void
+    {
+        $this->beConstructedWith([
+            'string' => "@SyliusUi/Grid/Filter/string.html.twig",
+            'boolean' => "@SyliusUi/Grid/Filter/boolean.html.twig",
+        ]);
+    }
+
+    function it_is_initializable()
```

Comment:
```suggestion
    function it_is_initializable(): void
```
or let's remove it. Same for the rest of the specs